### ISSUE TITLE
Specify a license under which this project can be used

### ISSUE THREAD

**Issue by Wulfheart** [leokhoa/laragon#166]
Hey,

please provide a License under which this project can be used to give developers safety in using this project.

Best regards

**Comment by h3rb** [comment]
Yes, if you wish to "give it away free" please choose MIT or compatible license so our legal department doesn't get mad and place the file in the README.md or in its own LICENSE file inside the repo...

**Comment by bliepp** [comment]
Not only a license would be great but also the source code (even if the license would not be a FOSS one). I guess this software would gain a lot when the community helps in development.

Additionally:
I am no lawyer, but since laragon **ships** with some copyleft licensed projects (namely HeidiSQL, MySQL, Notepad++) it may be seen as a derivative of these assets and may therefore be forced to be published under some FOSS license (I'm not talking about the option to download components later).

**Comment by leokhoa** [comment]
Laragon binary itself is a service orchestration tool that makes everything portable and reliable. It runs services asynchronously and non-blocking, so things feel fast and smooth. The open-source softwares (where licenses allow binary distribution) shipped with Laragon is to help you get started quickly. Laragon doesn't modify any source code of the softwares.

### In short:
1. **Before Laragon 7.0.0 (Released on Dec 16, 2024):** No license was needed! It was completely free as my goodwill to the community, so there was no risk in using it.  
2. **From Laragon 7.0.0 Onward:** You'll need to buy a license from the [Lemon Squeezy Store](https://laragon.lemonsqueezy.com) to use Laragon.
